### ISSUE TITLE
[BUG] Control + Shift + C does not work on Mac

### ISSUE THREAD

**Issue by im-alihashmi14** [Sxvxgee/UnlimitedGPT#33]
### Description
When attempting to use the key combination Control + Shift + C in a macOS environment while using UnlimitedGPT, the expected behavior of the key combination is not occurring.

### Code used
```python
def _get_new_response(self):
        body = self.driver.find_element(By.TAG_NAME, "body")
        body.send_keys(Keys.COMMAND, Keys.LEFT_SHIFT, "c")
        return pyperclip.paste()

```

### Checklist
- [x] I have reviewed the documentation before submitting this bug report.
- [x] I have confirmed that I am running the latest version of UnlimitedGPT.
- [x] I have verified that the issue is not caused by any modifications I made to the library or code.
- [x] I have confirmed that the issue is caused by the library itself and not other dependencies used.

Please add a check for MacOS to use Command instead of Control

**Comment by Sxvxgee** [comment]
Hi, thanks for making this issue. When I added that copy feature I knew MacOS users are gonna face troubles but I didn't know if they even had the same key shortcuts as on Windows, so I just didn't bother adding a check for it. I'll release a fix to it soon. Could you please tell me about other key shortcuts that you have on the website if there are ones different than Windows, besides using command instead of control?

**Comment by im-alihashmi14** [comment]
Thanks.
Actually I don't know any other keys. I just faced this issue so I searched the required keys for a copy. 
Whenever you implement some keys related feature try to add checks for other platforms too.

**Comment by Sxvxgee** [comment]
Fixed in `0.1.9.3`. Please update to the new version to fix this issue, and let me know if it still persists

**Comment by Sxvxgee** [comment]
Bug fixed.
